script.js

```diff
@@ -8,4 +8,138 @@ openMenu();
+    image: 'img/projects.png',
+    technologies: ['HTML', 'Bootstrap', 'Ruby on rails'],
+    liveLink: 'https://example.com/project2',
+    sourceLink: 'https://github.com/user/project2',
+  },
+  {
+    index: 1,
+    name: 'Multi Post Stories',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    image: 'img/projects.png',
+    technologies: ['HTML', 'Bootstrap', 'Ruby on rails'],
+    liveLink: 'https://example.com/project2',
+    sourceLink: 'https://github.com/user/project2',
+  },
+  {
+    index: 2,
+    name: 'Multi Post Stories',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    technologies: ['HTML', 'Bootstrap', 'Ruby on rails'],
+    image: 'img/projects.png',
+    liveLink: 'https://example.com/project1',
+    sourceLink: 'https://github.com/user/project1',
+  },
+  {
+    index: 3,
+    name: 'Multi Post Stories',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    technologies: ['HTML', 'Bootstrap', 'Ruby on rails'],
+    image: 'img/projects.png',
+    liveLink: 'https://example.com/project1',
+    sourceLink: 'https://github.com/user/project1',
+  },
+  {
+    index: 4,
+    name: 'Multi Post Stories',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    technoloBootstraps: ['HTML', 'Bootstrap', 'Ruby on rails'],
+    image: 'img/projects.png',
+    liveLink: 'https://example.com/project1',
+    sourceLink: 'https://github.com/user/project1',
+  },
```

Comment:
- [ ] As stated in the project requirement you should implement popups for  desktop screen sizes: So when I checked on the Pop up I got the following error on the console  **"Uncaught TypeError: project.technologies is undefined"** occurs because the technologies property is missing or undefined for one of the projects in the projects array. **Specifically, it seems that the technologies property is not defined for the 3rd project object and is also misspelled as technoloBootstraps in the 5th project object.** This causes the error when trying to access the project.technologies property in the openModal function.

- **To fix this issue, ensure that all project objects have the technologies property defined correctly. Also, correct the misspelling of technoloBootstraps to technologies in the 5th project object.** 
![pop up](https://github.com/Obimbo07/Portfolio-2/assets/93598189/d474a058-50c8-4250-bf53-3124ee00fb5f)